test_gvisor.py

```
@@ -12,7 +12,7 @@
 project_root = Path(__file__).parent
 sys.path.insert(0, str(project_root))
 
-from app.modules.utils.gvisor_runner import (
+from app.modules.utils.gvisor_runner import (  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)